.gitignore

```diff
@@ -30,7 +30,7 @@ docs/_build
 # Pycharm editor project files
 .idea
 
-# for virtual environment
```

Comment:
why are you removing it, can't understand

Reply:
I thought it was not useful. I should add it?

Reply:
just comments, there is no need to bother about them. Leave it unchanged